include/dlr.h

```diff
@@ -37,7 +41,7 @@ typedef void* DLRModelHandle;
 /*!
  \brief Creates a DLR model.
  \param handle The pointer to save the model handle.
- \param model_path Path to the top-level directory containing the model.
```

Comment:
I guess what we discussed was that we still support old approach (model_path = top-level directory containing the model) and in addition to it we support model_path string in format of colon-separated list of paths.

I expect to see changes in the implementations of
```
DLRBackend dlr::GetBackend(const std::string& dirname)
ModelPath dlr::GetTvmPaths(const std::string& dirname) 
ModelPath dlr::GetTreelitePaths(const std::string& dirname)
```
These method should check if `dirname` contains `:`

if NOT then these methods should use old implementations which uses `ListDir` to convert `dirname` to `paths_vec` vector.

If YES then they should split dirname using `:` into `paths_vec` vector.

Reply:
Production and consumption of path vector are separated so we only need to check for ':' in one centralized location, in CreateDLRModel(). 

Luckily ListDir() was able to handle multiple dirs in the first place, and both GetBackend() and GetTvmPaths() have been modified to accept path vector strings by looping on ListDir().

Thanks for catching the missing part in GetTreelitePaths(). I'll add corresponding change for consistency.